### PR TITLE
[CORL 133] API Review

### PR DESCRIPTION
**Pull request by wyattjoh**
## What does this PR do?

- Removed Management API
- Simplified Tenant APIs
- Added Stories edge
- Added Users edge
- Resolves permission issue with creating comment replies
- Removed display name, made username non-unique
- Improved API documentation

### CODE REVIEW

File: src/core/client/stream/test/comments/postLocalReply.spec.tsx [hunk 1]
```diff
@@ -31,26 +31,32 @@ beforeEach(() => {
...
-              ...baseComment,
-              id: "comment-x",
-              author: users[0],
-              body: "<b>Hello world! (from server)</b>",
-            },
-          },
-          clientMutationId: data.input.clientMutationId,
-        };
-      }),
+      createCommentReply: createSinonStub(
```
Comment (alinhle) [inline src/core/client/stream/test/comments/postLocalReply.spec.tsx]: This looks like a merge error, the removed side looks like the newer one.

Comment (alinhle) [reply src/core/client/stream/test/comments/postLocalReply.spec.tsx]: Except replacing `cursor: null` with `cursor: ""`

File: src/core/server/app/router/api/auth.ts [hunk 2]
```diff
@@ -33,23 +33,15 @@ export function createNewAuthRouter(app: AppOptions, options: RouterOptions) {
   const router = express.Router();
 
   // Mount the logout handler.
-  router.delete(
-    "/",
-    options.passport.authenticate("jwt", { session: false }),
```
Comment (alinhle) [inline src/core/server/app/router/api/auth.ts:38]: Just checking, this line is not required anymore?

File: src/core/server/graph/tenant/context.ts [hunk 3]
```diff
@@ -1,64 +1,52 @@
...
-export interface TenantContextOptions {
+export interface TenantContextOptions extends CommonContextOptions {
   mongo: Db;
   redis: AugmentedRedis;
   tenant: Tenant;
   tenantCache: TenantCache;
-  queue: TaskQueue;
-  config: Config;
+  mailer: MailerQueue;
+  scraper: ScraperQueue;
```
Comment (alinhle) [inline src/core/server/graph/tenant/context.ts]: Should these be `mailerQueue` and `scraperQueue`?

**Review by alinhle — CHANGES_REQUESTED** [review]
Looks good, just a view comments

File: src/core/client/stream/test/comments/postLocalReply.spec.tsx (hunk 1 above)
Comment (wyattjoh) [reply src/core/client/stream/test/comments/postLocalReply.spec.tsx]: I'm still a bit unclear, can you take a stab at resolving this one?

File: src/core/server/app/router/api/auth.ts (hunk 2 above)
Comment (wyattjoh) [reply src/core/server/app/router/api/auth.ts:38]: It's replaced with this one:

https://github.com/coralproject/talk/blob/792778818fdf4f7da8d58393040fd20fb5e441c6/src/core/server/app/router/api/index.ts#L46-L48

File: src/core/client/stream/test/comments/postLocalReply.spec.tsx (hunk 1 above)
Comment (alinhle) [reply src/core/client/stream/test/comments/postLocalReply.spec.tsx]: Sure!

**Review by alinhle — APPROVED** [review]

**Pull request merged**